cleanrl/sac_atari.py

```diff
@@ -0,0 +1,342 @@
+    parser.add_argument("--seed", type=int, default=1, 
+        help="seed of the experiment")
+    parser.add_argument("--torch-deterministic", type=lambda x: bool(strtobool(x)), default=True, nargs="?", const=True,
+        help="if toggled, `torch.backends.cudnn.deterministic=False`")
+    parser.add_argument("--cuda", type=lambda x: bool(strtobool(x)), default=True, nargs="?", const=True,
+        help="if toggled, cuda will be enabled by default")
+    parser.add_argument("--track", type=lambda x: bool(strtobool(x)), default=False, nargs="?", const=True,
+        help="if toggled, this experiment will be tracked with Weights and Biases")
+    parser.add_argument("--wandb-project-name", type=str, default="cleanRL",
+        help="the wandb's project name")
+    parser.add_argument("--wandb-entity", type=str, default=None,
+        help="the entity (team) of wandb's project")
+    parser.add_argument("--capture-video", type=lambda x: bool(strtobool(x)), default=False, nargs="?", const=True,
+        help="weather to capture videos of the agent performances (check out `videos` folder)")
+
+    # Algorithm specific arguments
+    parser.add_argument("--env-id", type=str, default="BeamRiderNoFrameskip-v4",
+        help="the id of the environment")
+    parser.add_argument("--total-timesteps", type=int, default=5000000,
+        help="total timesteps of the experiments")
+    parser.add_argument("--buffer-size", type=int, default=int(1e6),
+        help="the replay memory buffer size") # smaller than in original paper but evaluation is done only for 100k steps anyway
+    parser.add_argument("--gamma", type=float, default=0.99,
+        help="the discount factor gamma")
+    parser.add_argument("--tau", type=float, default=1.0,
+        help="target smoothing coefficient (default: 1)") # Default is 1 to perform replacement update
+    parser.add_argument("--batch-size", type=int, default=64,
+        help="the batch size of sample from the reply memory")
+    parser.add_argument("--learning-starts", type=int, default=2e4,
+        help="timestep to start learning")
+    parser.add_argument("--policy-lr", type=float, default=3e-4,
+        help="the learning rate of the policy network optimizer")
+    parser.add_argument("--q-lr", type=float, default=3e-4,
+        help="the learning rate of the Q network network optimizer")
+    parser.add_argument("--update-frequency", type=int, default=4,
+        help="the frequency of training updates")
+    parser.add_argument("--target-network-frequency", type=int, default=8000,
+        help="the frequency of updates for the target networks")
+    parser.add_argument("--alpha", type=float, default=0.2,
+        help="Entropy regularization coefficient.")
+    parser.add_argument("--autotune", type=lambda x:bool(strtobool(x)), default=True, nargs="?", const=True,
+        help="automatic tuning of the entropy coefficient")
+    parser.add_argument("--target-entropy-scale", type=float, default=0.9,
```

Comment:
The experiment results reported using `target-entropy-scale=0.89`, so maybe change it to 0.89 here to be consistent? 

<img width="602" alt="image" src="https://user-images.githubusercontent.com/5555347/212114620-63c37989-ed6e-4e43-8acc-b8ad7a660914.png">